ols/src/tools/oc_cli.py

```diff
@@ -31,16 +32,20 @@
 
 def strip_args_for_oc_command(args: list[str]) -> list[str]:
     """Sanitize arguments for `oc` CLI commands if LLM adds it extra."""
-    # Sometimes within the list we may get two args combined; ex: [top pod]
-    striped_args = " ".join(args).split(" ")
-    # Sometimes model gives args which are already added to the tool.
+    # sometimes within the list we may get two args combined, eg: [top pod]
+    splited_args = " ".join(args).split(" ")
+
+    # strip whitespaces from args
+    splited_args = [arg.strip() for arg in splited_args if arg.strip()]
```

Comment:
Based on [this](https://github.com/openshift/lightspeed-service/pull/2478/files#r2072737541) we can modify below logic
```suggestion
    # sometimes within the list we may get two args combined, eg: [top pod]
    splited_args = [arg for arg in " ".join(args).split(" ") if arg]
```